### PR TITLE
[fix ci] Disable macos tests for native_layer_norm_float32

### PR DESCRIPTION
**Pull request by shubhambhokare1**

### CODE REVIEW

File: tests/function_libs/torch_lib/ops_test_common.py [hunk 1]
```diff
@@ -56,7 +57,8 @@
 )
 
 TEST_OPSET_VERSION = 18
-IS_WINDOWS = os.name == "nt"
+IS_MACOS = sys.platform.startswith('darwin')
+IS_WINDOWS = sys.platform.startswith('win32')
```
Comment (justinchuby) [inline tests/function_libs/torch_lib/ops_test_common.py:61]: I think the original check for windows is reliable; or are there exceptions not handled?

File: tests/function_libs/torch_lib/ops_test_data.py [hunk 2]
```diff
@@ -1932,6 +1932,10 @@ def _where_input_wrangler(
         core_ops.aten_native_layer_norm,
         trace_only=True,
         tolerance={torch.float32: (3.7e-5, 1.8e-4), torch.float16: (1e-1, 7e-4)},
+    ).xfail(
+        dtypes=(torch.float32,),
```
Comment (justinchuby) [inline tests/function_libs/torch_lib/ops_test_data.py:1936]: I would check the input size. Only skip the size-1 inputs.

**Comment by justinchuby** [comment]
Thanks for creating the fix! Minor comments

File: tests/function_libs/torch_lib/ops_test_common.py (hunk 1 above)
Comment (shubhambhokare1) [reply tests/function_libs/torch_lib/ops_test_common.py:61]: Had changed the format for the sake of uniformity. But the previous check was sufficient. Reverting back to that.

**Review by justinchuby — APPROVED** [review]
Thank you!

**Pull request merged**